src/Drawer/Drawer.js

```diff
@@ -28,42 +29,42 @@ export const styleSheet = createStyleSheet('Drawer', (theme) => {
   };
 });
 
+type Props = {
+  /**
+   * The contents of the `Drawer`
+   */
+  children?: Object,
+  /**
+   * The CSS class name of the root element.
+   */
+  className?: string,
+  /**
+   * If set to true, the drawer will dock itself
+   * and will no longer slide in with an overlay
+   */
+  docked?: boolean,
+  open: boolean,
+  /**
+   * The CSS class name of the paper element.
+   */
+  paperClassName?: string,
+  zDepth: number,
```

Comment:
I'm pretty new to flowtype with react, but using a tool like react-docgen, would `zDepth` in this case be output as a required field? If so, even if we provide a default value, we may still need to put a `?` here (sorry normally I would try before asking but don't want to forget)